.eslintrc

```diff
@@ -4,14 +4,14 @@
     "sourceType": "module"
   },
   "plugins": ["jest-formatting", "jest"],
-  "rules": {
-    "jest-formatting/padding-before-test-blocks": 2,
-    "jest-formatting/padding-before-describe-blocks": 2
-  },
   "env": {
     "jest/globals": true
   },
-  "extends": ["airbnb-base", "plugin:prettier/recommended"],
+  "extends": [
+    "airbnb-base",
+    "plugin:prettier/recommended",
+    "plugin:jest-formatting/recommended"
```

Comment:
This isn't important, but I feel like we may not need the `plugin:` prefix. I'll check the docs in a bit.

Reply:
We do indeed need it. We don't need it for airbnb because their config is it's own package... if i understand correctly. All good.